libraries/core_libs/consensus/src/pbft/pbft_manager.cpp

```diff
@@ -1636,7 +1637,8 @@ void PbftManager::finalize_(SyncBlock &&sync_block, std::vector<h256> &&finalize
   }
 }
 
-bool PbftManager::pushPbftBlock_(SyncBlock &&sync_block, vec_blk_t &&dag_blocks_order) {
+bool PbftManager::pushPbftBlock_(SyncBlock &&sync_block, std::vector<std::shared_ptr<Vote>> &&cert_votes,
```

Comment:
Maybe we can create a new class inherited from `SyncBlock` and it will also include this cert_votes. I think it will be cleaner and easier to understand than this additional argument in many methods. 
Also maybe we should consider renaming `SyncBlock` to `FullPbftBlockPeriodData` or something simplier, but more accurate than current name

Reply:
There is definitely this naming mismatch of column period_data and SyncBlock. The two should have some common name. I am not sure what would be the best name for both.

Reply:
Done